.codeclimate.yml

```diff
@@ -0,0 +1,5 @@
+---
+version: '2'
+checks:
+  method-count:
+    enabled: false
```

Comment:
This should not be necessary...we likely just missed adding the general .codeclimate.yml to this.

Reply:
You can ignore codeclimate for this PR as their issues are probably too strict for us.

Reply:
ok, reverted this.